### DISCUSSION TITLE
Switch port

### DISCUSSION THREAD

**Discussion by FieryMewtwo** [Universal-Team/Universal-Updater#111] [Ideas]
Yes, I know that `hbappstore` is a thing, but it hasn't been updated for a while and only supports 4TU's catalog. I think a Switch port of UU would be **amazing**, and you'd at least get one user out of it (me)!

**Comment by NightScript370** [comment]
- The Switch is a completely different ecosystem from the Nintendo DS line
- Homebrew Development scene is still early, meaning that we'd always have to chase the latest trend
- CFW Scene on Switch is extremely limited, contrast to the DS (needing a flashcart) and even DSi/3DS's free entry points. You need to own a specific model and insert a jig. I think only Evie meets those requirements
   - Emulators aren't accurate for Homebrew Development

The above is why Universal-Team will unfortunately not make port of Universal-Updater for the Nintendo Switch. Even if all those conditions above were filled, though, we're extremely burnt out from developing Universal-Updater. Our next version should be 4.0.0, in which there would be support for the DS line and an entire rewrite. After that, though, we see no reason to continue developing. Stack has already taken a break, especially considering his work was stolen from him.

If you're willing to visualize a brand new single-screen UI and give Stack hardware for testing, then it's a maybe. But for now, absolutely not.

**Reply by FieryMewtwo** [reply]
Drafting a user interface now :)

**Comment by lifehackerhansol** [comment]
The Switch already has https://github.com/fortheusers/hb-appstore, and is already widely used. I don't see a need for this.

It's not updated much, but neither is UU anyway.

**Reply by lifehackerhansol** [reply]
Also, just request any apps you want on their end. They are pretty willing.
